annotation-server/src/medications/medications.module.ts

```diff
@@ -1,17 +1,14 @@
 import { HttpModule } from '@nestjs/axios';
 import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
 import { TypeOrmModule } from '@nestjs/typeorm';
 
+import { Medication } from './medication.entity';
 import { MedicationsController } from './medications.controller';
-import { Medication } from './medications.entity';
 import { MedicationsService } from './medications.service';
-import { MedicationsGroup } from './medicationsGroup.entity';
 
 @Module({
-    imports: [
-        HttpModule,
-        TypeOrmModule.forFeature([Medication, MedicationsGroup]),
-    ],
+    imports: [ConfigModule, HttpModule, TypeOrmModule.forFeature([Medication])],
```

Comment:
ConfigModule is set to global in `app.module.ts`, I am 90% sure you dont need to actually import it here

Reply:
good point! will fix.